.gitignore

```diff
@@ -292,3 +292,5 @@ nb-configuration.xml
 ### Release creation ###
 tools/build/releases/*
 !tools/build/releases/.gitkeep
+admin-dev/themes/new-theme/public/module.bundle.js
```

Comment:
Can you give us more information about why ignoring this file twice?
And public files must be added in git :)

Reply:
Didn't know about that one. I changed it